notebooks/generate_synthetic_data.ipynb

```diff
@@ -210,23 +54,24 @@
     "id": "5PlLJJuOI-0A"
    },
    "source": [
-    "I googled for tutorials and it seems like there's a Python library called Pillow for working with images, including adding text to images with fonts.\n",
-    "\n",
-    "Based on some tutorials, here's a working example using the font we downloaded."
```

Comment:
Do you have the links to these tutorials so I can include them in the resources list I am compiling?